docs/grover.rst

```diff
@@ -26,3 +76,5 @@ grove.amplification.grover
         :members:
         :undoc-members:
         :show-inheritance:
+
+.. [1] https://en.wikipedia.org/wiki/Grover's_algorithm#Algebraic_proof_of_correctness
```

Comment:
Please do not use Wikipedia as a source for mathematical proofs. Track down the origin or cite Nielsen & Chuang for such claims.

Reply:
Yep.